docs/source/release/v6.2.0/muon.rst

```diff
@@ -102,6 +102,11 @@ Improvements
 ############
 - Updated :ref:`LoadElementalAnalysisData <algm-LoadElementalAnalysisData>` algorithm to include Poisson errors for the counts data.
 
+Bug Fixes
```

Comment:
Please can you change this to Bugfixes? This is in keeping with the rest of the release notes.

Reply:
```suggestion
Bugfixes
```